src/soc_ifc/rtl/soc_ifc_fuse_reg.rdl

```diff
@@ -83,7 +83,7 @@ reg {
     Fuse hsm_id[32]=0;
 } fuse_idevid_manuf_hsm_id[4];
 reg {
-    desc = " Caliptra Boot Media Integrated mode usage only. SOCs that build with a Boot Media Independent profile don’t have to account for these fuses.
+    desc = " Caliptra Boot Media Integrated mode usage only. SOCs that build with a Boot Media Dependent profile don’t have to account for these fuses.
```

Comment:
Profiles shouldn't be mentioned at all. The profile concept was removed from the main spec.

Reply:
Vishal Mhatre let me know Firmware isn't using this register all currently. The description could be changed to RESERVED for future usage - or we can leave it, since this is a Hardware source file and we're so late stage on the release.